modules/dnn/src/darknet/darknet_io.cpp

```diff
@@ -505,16 +565,24 @@ namespace cv {
                         }
                         else {
                             // read section
+                            isDropout = false;
                             read_net = false;
-                            ++layers_counter;
                             const size_t layer_type_size = line.find("]") - 1;
                             CV_Assert(layer_type_size < line.size());
                             std::string layer_type = line.substr(1, layer_type_size);
+                            if(layer_type == "dropout")
```

Comment:
please include single `dropout` layer into your `connected` config file. So we could test both of them

Reply:
Done !